src/styles/calendar.scss

```diff
@@ -0,0 +1,34 @@
+.calendar {
+  margin: 100px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+  padding: $calendarPadding;
+  width: $calendarMaxWidth;
+
+  &__day {
+    height: $cellHeight;
+    box-sizing: border-box;
+    width: $cellHeight;
+    position: relative;
+    border: 1px solid black;
+    background-color: #eee;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: 0.5s;
+    }
+    
+    &::before {
+      display: flex;

```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)